### PR TITLE
Rename Fixers

### LINKED ISSUE #889: Improving Fixer Names In 2.0
Here are my ideas. Please discuss! :)

| Current Name                       | Proposed Name                      |
|------------------------------------|------------------------------------|
| eof_ending                         | single_eol_at_eof                  |
| function_call_space                | no_spaces_after_method_name        |
| function_declaration               | closure_definition_spacing         |
| linefeed                           | unix_line_endings                  |
| multiple_use                       | single_statement_per_import        |
| parenthesis                        | no_spaces_inside_parenthesis       |
| trailing_spaces                    | no_trailing_whitespace             |
| visibility                         | visibility_required                |
| double_arrow_multiline_whitespaces | double_arrow_multiline_whitespace  |
| duplicate_semicolon                | no_duplicate_semicolons            |
| multiline_array_trailing_comma     | trailing_comma_multiline_array     |
| namespace_no_leading_whitespace    | no_leading_namespace_whitespace    |
| single_array_no_trailing_comma     | no_trailing_comma_single_array     |
| multiline_spaces_before_semicolon  | no_whitespace_before_semicolons    |

### PR DESCRIPTION
**Pull request by keradus**
Closes #889

### CODE REVIEW

File: src/Fixer/PSR2/BracesFixer.php [hunk 1]
```diff
@@ -73,8 +73,8 @@ private function fixCommentBeforeBrace(Tokens $tokens)
                 continue;
             }
 
-            $parenthesisEndIndex = $this->findParenthesisEnd($tokens, $index);
-            $afterParenthesisIndex = $tokens->getNextNonWhitespace($parenthesisEndIndex);
+            $no_spaces_inside_parenthesisEndIndex = $this->findParenthesisEnd($tokens, $index);
```
Comment (keradus) [inline src/Fixer/PSR2/BracesFixer.php]: to be reverted

File: src/Fixer/PSR2/NoSpacesAfterFunctionNameFixer.php [hunk 2]
```diff
@@ -74,7 +74,7 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
      */
     public function getDescription()
     {
-        return 'When making a method or function call, there MUST NOT be a space between the method or function name and the opening parenthesis.';
+        return 'When making a method or function call, there MUST NOT be a space between the method or function name and the opening no_spaces_inside_parenthesis.';
```
Comment (keradus) [inline src/Fixer/PSR2/NoSpacesAfterFunctionNameFixer.php]: to be reverted

File: README.rst [hunk 3]
```diff
@@ -146,14 +146,14 @@ apply (the fixer names must be separated by a comma):
...
-    php php-cs-fixer.phar fix /path/to/dir --rules=linefeed,short_tag,indentation
+    php php-cs-fixer.phar fix /path/to/dir --rules=unix_line_endings,full_opening_tag,no_tabs
 
 You can also blacklist the fixers you don't want by placing a dash in front of the fixer name, if this is more convenient,
 using ``-name_of_fixer``:
 
 .. code-block:: bash
 
-    php php-cs-fixer.phar fix /path/to/dir --rules=-short_tag,-indentation
+    php php-cs-fixer.phar fix /path/to/dir --rules=-full_opening_tag,-no_tabs
```
Comment (SpacePossum) [inline README.rst]: suggestion `no_tabs` -> `no_tab_indentations`, as we do allow tabs for other reasons.

File: src/Fixer/PSR2/SingleBlankLineAtEofFixer.php [hunk 4]
```diff
@@ -16,14 +16,14 @@
 use Symfony\CS\Tokenizer\Tokens;
 
 /**
- * A file must always end with a linefeed character.
+ * A file must always end with a unix_line_endings character.
```
Comment (SpacePossum) [inline src/Fixer/PSR2/SingleBlankLineAtEofFixer.php]: suggestion `unix_line_endings` -> `UNIX line ending`

File: src/Fixer/Symfony/MethodSeparationFixer.php [hunk 5]
```diff
@@ -38,7 +38,7 @@ public function getDescription()
      */
     public function getPriority()
     {
-        // Must run before braces and indentation fixers because this fixer
+        // Must run before braces and no_tabs fixers because this fixer
```
Comment (SpacePossum) [inline src/Fixer/Symfony/MethodSeparationFixer.php]: maybe use the class name of the fixers like every else is done?

Comment (keradus) [inline src/Fixer/Symfony/MethodSeparationFixer.php]: All name-changes was made automatically. That is why there is some false-positive changes that should be reverted.
Some comments in `getPriority` uses Fixer names, some rule names. While unification is good, it is not scope of this PR.

Comment (keradus) [inline src/Fixer/Symfony/MethodSeparationFixer.php]: done

**Comment by keradus** [comment]
done

**Comment by SpacePossum** [comment]
thanks for updating,
 :+1:

**Comment by keradus** [comment]
Then, mergy mergy ? :)

**Comment by GrahamCampbell** [comment]
:ship:

**Comment by SpacePossum** [comment]
>Then, mergy mergy ? :)

yeah, like @GrahamCampbell said, ship it :)
:package:

**Pull request closed without merging**